playwright/all_tests/assignment/assignment.spec.ts

```diff
@@ -218,3 +218,23 @@ test.describe('Puhvi assignment form tests', () => {
     page.getByTestId('create-koetehtava-button')
   })
 })
+
+test.describe('Presentation view', () => {
+  const presentationViewAssignment = testAssignmentIn('Esitysnäkymätestitehtävä')
+
+  test('can navigate to presentation view', async ({ page, context, baseURL }) => {
+    const assignment = await createAssignment(context, baseURL!, presentationViewAssignment)
+    await page.goto(`/suko/assignments/${assignment.id}`)
+
+    await expect(page.getByTestId('assignment-header')).toHaveText(presentationViewAssignment.nameFi)
+    // Use non-waiting assert to ensure later similar assert on new tab works as expected
+    expect(await page.$('[data-testid="assignment-metadata"]')).not.toBeNull()
```

Comment:
Jos mahollista olisi hyvä välttää $ -selectorin käyttöä, se on deprekoitu. 
Lisää inffoa selectoreista (aika pohjalla on deprecated osio): 
https://playwright.dev/docs/api/class-page#page-query-selector

Reply:
Haa, näemmä `await expect(await newTabPage.getByTestId('assignment-metadata')).not.toBeVisible()` on fiksu eikä jää venailee, jos elementti puuttuu kokonaan. Eli en tarviikaa tota page.$